initfiles/bin/init_thorslave

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+################################################################################
+#    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+################################################################################
+
+ip=$1
+master=$2
+masterport=$3
+logpth=$4
+instancedir=$5
+deploydir=$6
+hpcc_compname=$7
+hpcc_setenv=$8
+logredirect=$9
+cmd=${10}
+
+source ${hpcc_setenv}
+
+stop_slaves()
+{
+    local slavename=thorslave_${hpcc_compname}
+
+    killall -0 $slavename > /dev/null 2>&1
+    if [[ $? -eq 0 ]];then
+        killall -9 $slavename > /dev/null 2>&1
+    fi
+    rm -f $PID/${hpcc_compname}_slave_*.pid > /dev/null 2>&1
+}
+
+start_slaves()
+{
+    # insuring dafilesrv is running on the machine as it is a prerequisite
+    sudo /etc/init.d/dafilesrv status > /dev/null 2>&1
+    if [[ $? -ne 0 ]];then
+    sudo /etc/init.d/dafilesrv start > /dev/null 2>&1
+        if [[ $? -ne 0 ]];then
+            exit 1
+        fi
+    fi
+
+    # insuring parent directory structure is setup properly
+    mkdir -p $instancedir
+    mkdir -p $(dirname $logredirect)
+    exec >>$logredirect 2>&1
+
+    cd $instancedir
+
+    echo "$(date) Dependency dafilesrv is running"
+    echo "$(date) slave(${ip}) init"
+
+    lckfile="$PID/start_slaves_${hpcc_compname}_${ip}.pid"
+
+    # prevent two of these scripts starting together
```

Comment:
Yes, Had it commented out for testing purposes.  I'll be removing it completely.